tests/Unit/Core/Logger/LoggerManagerTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+// Copyright (C) 2010-2024, the Friendica project
+// SPDX-FileCopyrightText: 2010-2024 the Friendica project
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+declare(strict_types=1);
+
+namespace Friendica\Test\Unit\Core\Logger;
+
+use Friendica\Core\Config\Capability\IManageConfigValues;
+use Friendica\Core\Logger\Capability\LogChannel;
+use Friendica\Core\Logger\Factory\LoggerFactory;
+use Friendica\Core\Logger\LoggerManager;
+use Friendica\Core\Logger\Type\ProfilerLogger;
+use Friendica\Core\Logger\Type\WorkerLogger;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+
+class LoggerManagerTest extends TestCase
+{
+	public function testGetLoggerReturnsPsrLogger(): void
+	{
+		$reflectionProperty = new \ReflectionProperty(LoggerManager::class, 'logger');
+		$reflectionProperty->setAccessible(true);
+		$reflectionProperty->setValue(null, null);
```

Comment:
Why is this necessary? It doesn't look like the property is used directly in the test method.

Reply:
This code is needed because `LoggerManager::$logger` is static. For the test this propery should be `null` but it could be already set by a previous test. Thats the downside of the workaround (and the reason why the singleton pattern is an anti-pattern.

Maybe I should move the "cleaning" at the end of every test method to make this more clear.

Reply:
I moved the cleaning into the `tearDown()` method and added the cleaning of `$logChannel`.